geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -59,7 +59,7 @@ def init_params_and_state(self):
 
         :return: tuple(params, state)
         """
-        params = dict(lengthscale=1.0, nu=0.5)
+        params = dict(lengthscale=np.r_[1.0], nu=np.r_[0.5])
```

Comment:
Should we be using np directly? Is there a LAB call to make array instead?

Reply:
It's okay, lab is essentially a wrapper and everything could be converted from numpy.